openwillis/measures/video/preprocess_video.py

```diff
@@ -410,7 +410,7 @@ def extract_embed_faces_from_frame(
         face_data_list = prep_face_data_for_embed(face_data_list, frame)
         face_data_list = embed_faces(face_data_list, model_name=model_name)
     except Exception as e:
-        logger.info(f"Error extracting and embedding faces: {e}, frame_idx: {frame_idx}")
+        logger.error(f"Error extracting and embedding faces: {e}, frame_idx: {frame_idx}")
```

Comment:
Let's make this `logger.info` again, to have it consistent with the rest of the repo.
```suggestion
        logger.info(f"Error extracting and embedding faces: {e}, frame_idx: {frame_idx}")
```